crates/modelardb_storage/src/data_folder.rs

```diff
@@ -874,7 +873,7 @@ impl DataFolder {
             .await
     }
 
-    /// Write `record_batches` with segments to a Delta Lake table for a time series table with
+    /// Write `compressed_segments` with segments to a Delta Lake table for a time series table with
```

Comment:
Maybe also remove "with segments" since it is already clear that it is with segments in `compressed_segments`.

Reply:
Fixed.